packages/examples/examples/webpack/webpack.config.ts

```diff
@@ -5,9 +5,6 @@ import { Configuration } from 'webpack';
 import { merge } from 'webpack-merge';
 import WebpackBarPlugin from 'webpackbar';
 
-// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
-const HookShellScriptWebpackPlugin = require('hook-shell-script-webpack-plugin');
```

Comment:
Can we wait with doing this until we add the functionality to the plugin?

Reply:
It was breaking for some reason, I guess because `console.warn` was called? So I just removed it for now.